src/Zope2/App/tests/test_startup.py

```diff
@@ -74,6 +74,4 @@ def test_interface(self):
 
 
 def test_suite():
-    suite = unittest.TestSuite()
-    suite.addTest(unittest.makeSuite(StartupTests))
-    return suite
+    return unittest.defaultTestLoader.loadTestsFromTestCase(StartupTests)
```

Comment:
If there are only unittests collected in the `test_suite` method, it can be completely removed as `zope.testrunner` has a mechanism to detect unittest tests automatically.

Reply:
It is not that simple. Removing actually ends up with **more** tests run because all of a sudden e.g. base classes get run as well. I saw that with tests that use `Testing.ZopeTestCase`. Figuring that out is a job for another day.